### ISSUE TITLE
cli update from CRAN

### ISSUE THREAD

**Issue by gaborcsardi** [r-lib/cli#654]
```
Version: 3.6.1
Check: whether package can be installed
Result: WARN 
  Found the following significant warnings:
    progress.c:98:11: warning: too many arguments for format [-Wformat-extra-args]
  See ‘/home/hornik/tmp/R.check/r-devel-gcc/Work/PKGS/cli.Rcheck/00install.out’ for details.
  * used C compiler: ‘gcc-13 (Debian 13.2.0-5) 13.2.0’
Flavor: [r-devel-linux-x86_64-debian-gcc](https://www.r-project.org/nosvn/R.check/r-devel-linux-x86_64-debian-gcc/cli-00check.html)
```

```
Version: 3.6.1
Check: whether package can be installed
Result: WARN 
  Found the following significant warnings:
    utils.c:15:6: warning: function declaration isn't a prototype [-Wstrict-prototypes]
  See 'd:/Rcompile/CRANpkg/local/4.4/cli.Rcheck/00install.out' for details.
  * used C compiler: 'gcc.exe (GCC) 12.3.0'
Flavor: [r-devel-windows-x86_64](https://www.r-project.org/nosvn/R.check/r-devel-windows-x86_64/cli-00check.html)
```

**Comment by gaborcsardi** [comment]
Issue fixed in d8883112a8060c1f011cccc9147dd35db03bb9c6 and 667af285ba266491f8f8ebfdab9684e25062f271

**Comment by gaborcsardi** [comment]
Fixed in #659.
